source/_posts/2016-11-21-are-you-a-good-person.markdown

```diff
@@ -0,0 +1,130 @@
+is blindness. If that makes you feel bad about yourself, take some time to
+process that emotion and then move past it. We'll circle back to the whole "good
+person" thing here in a sec.
+
+
+## Identities are Bullshit
+
+I am not a feminist. You are not an ally. A book is not sexist. A movie is not
+racist. These phrases are simplifications and abstractions that don't really buy
+us much, but can lead to lazy thinking. Melissa McEwan has a wonderful
+[post on the topic](http://www.shakesville.com/2013/04/on-fixed-state-ally-model-vs-process.html).
+What matters is not whether I am an ally, but whether I _do ally work_.
+
+I started thinking this way first about myself and allyship and then about other
+things, specifically the media I consume. At first, I got hung up on whether a
+work "was feminist" or not. I was trying to ascribe some kind of Social Justice
+Score™ to everything. This was constantly thwarted by media that were very
+mixed. One episode of a show would have this wonderful pro-woman message and
+then the next would trot out a hackneyed misogynistic trope.
+
+I realized (with some external help as basically always) that my thinking about
+this phenomenon was all wrong. What matters is not what the show _is_, but what
+it _does_. How much feminist work does this show do? How much racist work does
+it do? These two answers can exist in parallel and allow for a much more robust
+criticism.
+
+So let's get back to the, "He may be a racist, but I'm not," thing. This
+importance of action over identity is the mechanism by which racism is
+communicable. Donald Trump does racist work. He takes actions that have racist
+outcomes. A person's support of him maginfies the effects of his work _including
+the racist work_. So supporting him is racist work. This is true regardless of
+what is in your heart or what reasoning you have.
+
+
+## You Must Observe Yourself to Discover Your Priorities
+
+One thing I've long held is that a verbal statement of priority means basically
+nothing. You don't set priorities by declaring them, you _enact_ them by _doing
+things_. Changing your habitual actions is a lot harder than saying you will
+change them and introspecting is a lot more challengeing than even that. But if
+you want to be real with yourself (and I think we all should want this), then
+it is necessary.
+
+The strategy I've taken to using is that I inspect the actual actions I take in
+the world and use that to _find out_ what my priorities are. It's one thing to
+say, "School work is a high priority for me," and another to actually set aside
+the time to study and do the work. I did not, it turns out, make school work a
+high priority when I was in college.
+
+
+So when you do something that constitues racist work (and let's give you the
+benefit of the doubt that it does _other_ work that is beneficial), you are
+making a clear, unambiguous statement of priority. You are saying that the
```

Comment:
"Statement" is confusing, here. Use "expression" or similar and reword accordingly.